pkg/skaffold/schema/latest/config.go

```diff
@@ -450,12 +450,15 @@ type ResourceRequirement struct {
 	ResourceStorage string `yaml:"resourceStorage,omitempty"`
 }
 
-// TestCase is a list of structure tests to run on images that Skaffold builds.
+// TestCase is a list of tests to run on images that Skaffold builds.
 type TestCase struct {
 	// ImageName is the artifact on which to run those tests.
 	// For example: `gcr.io/k8s-skaffold/example`.
 	ImageName string `yaml:"image" yamltags:"required"`
 
+	// CustomTests runs a custom test script provided by the user.
```

Comment:
```suggestion
	// CustomTests runs a set of custom test scripts provided by the user.
```
since `CustomTests` is a slice.